src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -258,6 +267,14 @@ namespace AirflowNetwork {
             Num
         };
 
+        enum class DuctSizeMethod
+        {
+            None = -1,
+            MaxVelocity,
+            PressureLoss,
+            VelocityAndLoss
+        };
+
         // Members
         std::string AirflowNetworkSimuName; // Provide a unique object name
         std::string Control;                // AirflowNetwork control: MULTIZONE WITH DISTRIBUTION,
```

Comment:
Not part of this PR, I know but since I can see it, I will say something. This should be an `enum class` rather than a string and there probably already is an `enum class` for it.

Reply:
Those changes are in #9449. Switching this to an enum was a little bit more involved that some of the others, there were some checks that involved arithmetic ( `value > MyGlobalInteger +1` and such) that needed to be reinterpreted. I have asked that this PR get merged first and then I'll deal with the inevitable conflicts and not hold up this feature.

Reply:
I leave it for the time being.